lib/transmit.js

```diff
@@ -285,6 +285,11 @@ internals.transmit = function (response, callback) {
         }
     }
 
+    const isInjection = Shot.isInjection(request.raw.req);
+    if (!isInjection && !request.connection._started) {
```

Comment:
This is a Hapi connection property lookup to know when `server.stop()` has been called.